src/rust/analyzer-dispatcher/src/main.rs

```diff
@@ -98,11 +94,14 @@ impl AnalyzerDispatcherConfig {
 
 #[derive(Debug, Error)]
 enum ConfigurationError {
-    #[error("error configuring kafka client")]
+    #[error("error configuring kafka client {0}")]
     KafkaConfiguration(#[from] KafkaConfigurationError),
 
-    #[error("error configuring analyzer IDs cache")]
-    AnalyzerIdsCacheConfiguration(#[from] ConnectError),
+    #[error("error configuring gRPC client {0}")]
+    ClientError(#[from] ClientError),
+
+    #[error("figment error {0}")]
+    FigmentError(#[from] figment::Error),
```

Comment:
i'd argue that a figment configuration error is a valid scenario for a panic - that's never recoverable - but this is fine